wordpress-mobile/WordPress-FluxC-Android
# Update encrypted test secrets to update TEST_WPCOM_IMAGE_IDS_TEST1

We have been having some intermittent failures in our connected tests. I've investigated the issue today and realized that the site we have been using became so bloated that `wp-admin` was having a lot of issues as well. Any bulk action I made, whether from classical or new admin or directly using the API had a lot of timeouts and random errors. To address the issue, I emptied the site which seems to have fixed the issues we were having in `wpcom` connected tests.

**Note that `ReleaseStack_WCProductTest.testUpdateProductPassword` is still failing, but it's not relevant to the changes in this PR. I'll raise this on Slack, so we can get someone from WCAndroid to take a look at it.**

It's unlikely that we'll need to do this again anytime soon, but in case we need to in the future, here are the steps I took:
1. From the WordPress.com `network-admin`, I found our testing site `fluxctestsite123.wordpress.com` and triggered the `Empty` action
2. Uploaded a new image to the site's Media tab and found its `id`
3. Updated the `TEST_WPCOM_IMAGE_IDS_TEST1` in our encrypted secrets for FluxC
4. Created a PR to update the `.configure` file and the encrypted files

**To test**

No manual testing is necessary, this PR simply updates the `TEST_WPCOM_IMAGE_IDS_TEST1` to the new value. However, if you're curious, you can compare the [Optional Firebase Test Results from this PR](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.1eb6894c0ff11a4e/matrices/5734541828924918554/details?stepId=bs.a5d51f5aa33ee885&testCaseId=595) with the latest [CircleCI FluxC Tests from yesterday's `trunk`](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.fbe4c2866a46a076/matrices/9145005831229576635/details?stepId=bs.726c343ba241cb36&testCaseId=410). Notice that `testUpdateProductPassword` fails in both runs, but `testAutoSaveLocalPostResultsInUnknownPostError`, `testClearTagsFromPost` & `testCancelImageUpload` were successful in this PR.

Please disregard the run from current `trunk` as it happened after I emptied the test site and before this PR is merged.
